defaults/main.yml

```diff
@@ -1,12 +1,23 @@
 ---
+openio_kibana_namespace: "OPENIO"
```

Comment:
openio_kibana_namespace: "{{ namespace | default('OPENIO') }}"

Reply:
fixed in the 3 roles